packages/manager/apps/vrack-services/src/api/order/hook.ts

```diff
@@ -46,12 +47,11 @@ export enum OrderDescription {
 }
 
 export const getDeliveringOrderQueryKey = (description: OrderDescription) => [
-  `deliveringOrders-${description}`,
+  'deliveringOrders',
+  description,
 ];
 
-export const getDeliveringOrderList = (
-  orderDetailDescription: string,
-) => async () => {
+const getDeliveringOrderList = (orderDetailDescription: string) => async () => {
```

Comment:
Don't understand why you need a function that return another one ?

Reply:
Fixed thanks